Eastern-Research-Group/mywaterway
# feature/refresh state intro text

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3163120

## Main Changes:
* Update the state intro text file with the most recent changes to the spreadsheet

## Steps To Test:
1. Navigate to state page and check the modified states
2. You might need to clear your cache to test this change. the file was cached locally for me
